packages/paste-core/primitives/combobox/stories/index.stories.tsx

```diff
@@ -80,14 +78,14 @@ export const AutocompleteCombobox = (): React.ReactNode => {
       }
     },
   });
-  const uid = useUID();
+  const seed = useUIDSeed();
   return (
     <>
-      <Label htmlFor={uid} {...getLabelProps()}>
+      <Label htmlFor={seed('autocomplete-combobox')} {...getLabelProps()}>
         Choose a component:
       </Label>
       <Box display="flex" {...getComboboxProps()}>
-        <Input id={uid} type="text" {...getInputProps()} />
+        <Input id={seed('autocomplete-combobox')} type="text" {...getInputProps()} />
```

Comment:
Random thought: I think downshift might be overriding the ids because theyre set before we pass `{...getInputProps()}`

Reply:
Oh good catch! Yeah we can actually pass a few diff config options to the `useCombobox` hook to either prefix each generated ID with our custom string, or specifically set an id (e.g. input id versus menu id).

I'll check out both when I switch back to this branch and update. 👍